### PR TITLE
Add function getWindowCentreInPixels

### PR DESCRIPTION
**Pull request by tischi**

### CODE REVIEW

File: src/main/java/sc/fiji/bdvpg/bdv/BdvHandleHelper.java [hunk 1]
```diff
@@ -29,34 +29,24 @@
...
-import bdv.ui.SourcesTransferable;
 import bdv.util.BdvHandle;
-import bdv.util.ResampledSource;
 import bdv.viewer.Source;
-import ch.epfl.biop.bdv.select.SourceSelectorBehaviour;
 import net.imglib2.*;
 import net.imglib2.realtransform.AffineTransform3D;
 import net.imglib2.util.Intervals;
 import net.imglib2.util.LinAlgHelpers;
+import org.jetbrains.annotations.NotNull;
```
Comment (NicoKiaru) [inline src/main/java/sc/fiji/bdvpg/bdv/BdvHandleHelper.java]: Can this be removed ?

**Review by NicoKiaru — COMMENTED** [review]
Just an extra import that should be removed. All good for me otherwise

File: src/main/java/sc/fiji/bdvpg/bdv/BdvHandleHelper.java (hunk 1 above)
Comment (tischi) [reply src/main/java/sc/fiji/bdvpg/bdv/BdvHandleHelper.java]: done

**Comment by tischi** [comment]
I added one more function, OK?

**Pull request merged**